planner/core/common_plans.go

```diff
@@ -834,8 +834,8 @@ func (h *AnalyzeTableID) Equals(t *AnalyzeTableID) bool {
 	return h.TableID == t.TableID && h.PartitionID == t.PartitionID
 }
 
-// analyzeInfo is used to store the database name, table name and partition name of analyze task.
-type analyzeInfo struct {
+// AnalyzeInfo is used to store the database name, table name and partition name of analyze task.
+type AnalyzeInfo struct {
```

Comment:
It seems that these lines will conflict with https://github.com/pingcap/tidb/pull/24999/files#diff-39be1be8697f0f187228689d867a84cb046ab2ff8ac643ac807244cdb7e2b842L837.

Reply:
This is the same change actually(